### DISCUSSION TITLE
关于进度条的问题

### DISCUSSION THREAD

**Discussion by fqh1995** [xiaye13579/BBLL#456] [Ideas]
官方云视听tv端可以支持正常播放下面显示进度条略缩。希望后期可以跟进
这款软件很棒了，很好给力

**Comment by digitalbaka** [comment]
支持你！


用人不宜刻，刻则思效者去；交友不宜滥，滥则贡谀者来。
我的个人空间bilibili
在2023年03月15日 10:12，fqh1995 写道：

官方云视听tv端可以支持正常播放下面显示进度条略缩。希望后期可以跟进
这款软件很棒了，很好给力
